### PR TITLE
Dosc update: Time formatter description added

### PR DESCRIPTION
**Pull request by adubrovnik**
**Type of PR:** 

**PR checklist:**

- [ ] Addresses an existing issue: fixes #
- [ ] Includes tests
- [x] Documentation update

**Overview of change:**

**Is there anything you'd like reviewers to focus on?**

### CODE REVIEW

File: docs/customization.md [hunk 1]
```diff
@@ -19,34 +73,47 @@ The following date formats are available:
...
-    height: 150,
+       localization: {
+           dateFormat: 'MM/dd/yy',
+       },
+});
+```
+
+#### Time Format
+
+`timeFormatter` function can be used to customize the format of time data displayed by vertical crosshair line.
```
Comment (ezhukovskiy) [inline docs/customization.md]: `timeFormatter` function can be used to customize the format of the time stamp displayed on the time axis below the vertical crosshair line.

File: docs/customization.md [hunk 2]
```diff
@@ -19,34 +73,47 @@ The following date formats are available:
...
+Currently, setting desirable time format on the time scale itself is not available, yet this feature is planned for the future.
```
Comment (ezhukovskiy) [inline docs/customization.md]: Currently, changing the time format of the time scale labels itself is not available, yet this feature is planned for the future.

File: docs/customization.md [hunk 3]
```diff
@@ -2,10 +2,55 @@
...
 
-Also, the `localization` option may help you set a suitable date format and user language.
-The following date formats are available:
+### Size
+
+First of all, the preferred chart size should be set when creating a chart:
+
+```javascript
+const chart = createChart(document.body, {
+    width: 600,
```
Comment (timocov) [inline docs/customization.md]:
```suggestion
        width: 600,
```

File: docs/customization.md [hunk 4]
```diff
@@ -2,10 +2,55 @@
...
+    height: 380,
```
Comment (timocov) [inline docs/customization.md]:
```suggestion
        height: 380,
```

File: docs/customization.md [hunk 5]
```diff
@@ -2,10 +2,55 @@
...
+chart.resize(250, 150);
+```
+
+### Localization
+
+Using the `localization` option you can set the displayed language, date and time formats.
+
+#### Locale
+
+By default, the library uses browser language settings. Thus, the displayed date and time format may differ depending on the region of the user. To set the same language settings for all users, use the `locale` property of the `localization` option:
```
Comment (timocov) [inline docs/customization.md]:
```suggestion
By default, the library uses browser language settings.
Thus, the displayed date and time format may differ depending on the region of the user.
To set the same language settings for all users, use the `locale` property of the `localization` option:
```

File: docs/customization.md [hunk 6]
```diff
@@ -2,10 +2,55 @@
...
+chart.applyOptions({
+   localization: {
+        locale: 'en-US',
+   },
+});
+```
+
+#### Date Format
+
+Preferred date format can be set using the `dateFormat` property of of the `localization` option. The following date formats are available:
```
Comment (timocov) [inline docs/customization.md]:
```suggestion
Preferred date format can be set using the `dateFormat` property of the `localization` option. The following date formats are available:
```

**Pull request merged**